source/renderer/app/components/status/DaedalusDiagnostics.js

```diff
@@ -293,13 +331,23 @@ const messages = defineMessages({
   },
   statusOn: {
     id: 'daedalus.diagnostics.dialog.statusOn',
-    defaultMessage: '!!!YES',
-    description: 'YES',
+    defaultMessage: '!!!Yes',
```

Comment:
Requirement from @alexander-rukin to stop using uppercase on diagnostics screen.